x-pack/plugins/observability/public/pages/slos/components/slo_list_item.tsx

```diff
@@ -161,11 +163,13 @@ export function SloListItem({
             </EuiFlexItem>
 
             <EuiFlexItem grow={false}>
-              <SloSummary
-                slo={slo}
-                historicalSummary={historicalSummary}
-                historicalSummaryLoading={historicalSummaryLoading}
-              />
+              {slo.summary ? (
```

Comment:
We don't have a proper loading state for the SloSummary yet, so decided to not render until we have a `summary`.

Reply:
Yeah, the problem here is that when we use an optimistic update, the fictive SLO object that gets put in the fetchSloList store does not have a summary object. The lacking of this object crashes this component as it expects a full SLO object and the fictive one isn't that.

I think we need to have a bigger think about this issue. Perhaps for 8.9?